portal/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+// This file is for typechecking only.
+// Parcel uses Babel to compile our TypeScript source files.
+// So here we only include options that are related to typechecking.
+{
+  "compilerOptions": {
```

Comment:
Since this should be used for typechecking only, let's add `noEmit` to tsconfig directly?